stacks/ubi8/base/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM registry.access.redhat.com/ubi8/ubi:8.3
+FROM registry.access.redhat.com/ubi8/ubi:8.4
 
 ARG cnb_uid=1000
 ARG cnb_gid=1001
 
 # Create user and group
 RUN groupadd -g ${cnb_gid} cnb && \
-  useradd -u ${cnb_uid} -g cnb -s /bin/bash cnb
+  useradd -u ${cnb_uid} -g cnb -s /bin/bash cnb && \
+  dnf install --nodocs --best -y git
```

Comment:
If the other dnf install commands needed --skip-broken, is it worth adding here as well?

Reply:
Removed - this was a hacky way to address some issues seen with the bump to 8.4 (those issues are also now resolved)